packages/data-stores/src/auth/actions.ts

```diff
@@ -131,36 +142,27 @@ export function createActions( {
 		const username = yield select( STORE_KEY, 'getUsernameOrEmail' );
 
 		try {
-			const loginResponse = yield fetchAndParse(
-				// TODO Wrap this in `localizeUrl` from lib/i18n-utils
-				'https://wordpress.com/wp-login.php?action=login-endpoint',
-				{
-					credentials: 'include',
-					method: 'POST',
-					headers: {
-						Accept: 'application/json',
-						'Content-Type': 'application/x-www-form-urlencoded',
-					},
-					body: stringify( {
-						remember_me: true,
-						username,
-						password,
-						client_id,
-						client_secret,
-					} ),
-				}
-			);
+			const loginResponse = yield* wpLogin( 'login-endpoint', {
+				remember_me: true,
+				username,
+				password,
+			} );
 
-			if ( loginResponse.ok && loginResponse.body.success ) {
-				if ( loadCookiesAfterLogin ) {
-					yield reloadProxy();
-					// Need to rerequest access after the proxy is reloaded
-					yield requestAllBlogsAccess();
+			if ( ! loginResponse.success ) {
+				yield receiveWpLoginFailed( loginResponse );
+			} else if ( loginResponse.data.two_step_notification_sent ) {
+				yield receiveWpLogin( loginResponse );
+
+				const twoFactorResonse = yield* handle2fa( loginResponse.data );
```

Comment:
Tiny typo! Should be `twoFactorResponse` not `twoFactorResonse` 😄

Reply:
Thanks 😆 all the words start melding together after a while